test-apps/ui-test-app/package.json

```diff
@@ -62,14 +62,10 @@
     "@types/react-beautiful-dnd": "^12.1.2",
     "cpx": "^1.5.0",
     "cross-env": "^5.1.4",
-    "dotenv": "^8.2.0",
```

Comment:
This should still be used by the backend code, even though the frontend has it bundled.

Reply:
re-added, nice catch. thx